packages/host/app/components/matrix/room.gts

```diff
@@ -117,7 +116,11 @@ export default class Room extends Component<Signature> {
   @service private declare matrixService: MatrixService;
   @service private declare operatorModeStateService: OperatorModeStateService;
 
-  private roomResource = getRoom(this, () => this.args.roomId);
+  private roomResource = getRoom(
+    this,
+    () => this.args.roomId,
+    () => this.matrixService.getRoom(this.args.roomId)?.events,
```

Comment:
why do we pass in events to the resource, can't the resouce look it up via the matrix service and the roomId that you supplied above? seem slike you are doing the work for the resource on the outside of the resource

Reply:
So I noticed that we're actually already fetching the roomResource for this component in `panel.gts` where we render this room component. I wonder if we should just pass it on to this component. If we're planning on moving away from fetching it in the panel component, however, then it's fine to do it here.